package.json

```diff
@@ -3,16 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.5.0",
+    "@emotion/styled": "^11.3.0",
+    "@mui/material": "^5.0.4",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
-    "bootstrap": "^4.5.2",
+    "@types/axios": "^0.14.0",
+    "@types/react-bootstrap": "^0.32.28",
+    "@types/react-redux": "^7.1.20",
+    "@types/redux": "^3.6.0",
+    "@types/redux-thunk": "^2.1.0",
+    "axios": "^0.24.0",
+    "bootstrap": "5.1.3",
+    "connect-api-mocker": "^1.10.0",
     "node-sass": "^4.14.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "react": "^17.0.2",
+    "react-bootstrap": "^2.0.0",
```

Comment:
Don't mix mui and react-bootstrap.

Please remove one.

Reply:
fixed